src/components/dynamic/admin/services/calendar/Events.jsx

```diff
@@ -25,6 +25,20 @@ const CalendarEvents = () => {
       case "w":
         setView("week");
         break;
+      case "ArrowRight":
+        if (view === "month") {
+          setDate(new Date(date.setMonth(date.getMonth() + 1)));
```

Comment:
should just be able to call event.NEXT, if you check the toolbar component you can see the way to change it